backend/fms_core/models/container.py

```diff
@@ -17,7 +17,6 @@
 from ._utils import add_error as _add_error
```

Comment:
Are those changes necessary? I think it would be easier to have it done later if it's simple string formatting

Reply:
They are not necessary but they are done ... i'll deal with the pain of the merge ... I don't see me gaining anything by reverting those changes ... I kind of thought about it once it was done and i thought as you did but in the end it will clean the code a bit and i was going through the code line by line which does not happen often... Anyway ... unless you think these are terrible edits i'll let them stand ... You can point out if some of them you don't agree with but i do not think i want to roll all of them back.

Reply:
that's ok for now since its done, I mean, it's just that you will probably merge your branch into the base branch before me and Sebastian does it, so we will have to deal with the merges from our branches then

Reply:
yeah i'll merge after you guys ...

Reply:
I need your part to have it merged before mine so the models can work :P thats ok, I will solve the conflicts on my branch after